### ISSUE TITLE
(aug-)cc-pVnZ-PP basis sets are not built-in

### ISSUE THREAD

**Issue by TiborGY** [psi4/psi4#2697]
The cc-pVnZ-PP and aug-cc-pVnZ-PP series of basis sets, optimized for heavy elements are not available as built in basis sets.
These basis sets are available for most elements from Y to Rn, with the exceptions of alkali and alkaline earth metals, and lanthanides.

They are optimized to be used with a small-core ECP that replaces only the inner core electrons, typically one from the Stuttgart-Köln ECP family.

Beyond just adding the relevant basis set and ECP definitions there is also a question of ECP selection. Based on a quick look at the docs, currently ECP-optimized basis sets and ECPs are somewhat tied together. The (aug-)cc-pVnZ-PP series does not really fit into that picture, as there are multiple valid choices of ECP. So there should be some user-facing option for selecting the ECP.

Also, many of the elements may not have appropriate auxiliary basis sets available, so if DF is the default for some methods, then they should probably switch to conventional/direct/incore instead of erroring out.

**Comment by hokru** [comment]
There isn't a single defined ECP that goes with the original basis set proposal? Computational detail sections I see often just specify PP basis set and not explicitly the ECP.

**Comment by TiborGY** [comment]
> There isn't a single defined ECP that goes with the original basis set proposal? Computational detail sections I see often just specify PP basis set and not explicitly the ECP.

I would have to look up the papers. Either there was none, or the ECPs have proliferated since or something. Molpro have had multiple Stuttgart-Köln ECPs to choose from, since a long time.

**Comment by JGrantHill** [comment]
The basis sets are specifically matched to ECPs, which is the appropriate ECPXXMDF for whichever row of the periodic table.

I also note that these basis sets **are** available for alkali and alkaline earth too.

**Comment by TiborGY** [comment]
Yes indeed ECPXXMDF is what we have been using with Molpro, and I think that is also what Orca 5 uses. The reason why I was not 100% sure is because the Molpro manual also discusses the MHF and MWB versions of these ECPs.

And there is also the question of small/large-core ECPs. While I think most people would probably use the small-core ones, that is yet another choice. (eg. replace 28 or 46 e- on iodine)

Thanks for the alkali(ne) correction, I only looked at what is listed on the BSE website.
